packages/grafana-ui/src/components/Tabs/Tab.tsx

```diff
@@ -88,42 +99,25 @@ const getTabStyles = stylesFactory((theme: GrafanaTheme2) => {
         color: ${theme.colors.text.primary};
 
         &::before {
-          display: block;
-          content: ' ';
-          position: absolute;
-          left: 0;
-          right: 0;
-          height: 4px;
-          border-radius: 2px;
-          bottom: 0px;
-          background: ${theme.colors.action.hover};
+          background-color: ${theme.colors.action.hover};
         }
       }
     `,
     activeStyle: css`
       label: activeTabStyle;
       color: ${theme.colors.text.primary};
       overflow: hidden;
-      font-weight: ${theme.typography.fontWeightMedium};
```

Comment:
This is the fix itself

Reply:
The other option would be to have all be 500 weight

Reply:
True, didn't think of that option. Here are screenshots for comparison:
![image](https://user-images.githubusercontent.com/3773351/222730917-2363959d-c9b8-414f-b059-c85ce47f4070.png)
![image](https://user-images.githubusercontent.com/3773351/222731098-d53b7544-167a-4aa1-a531-3fc1417dd53c.png)

Reply:
I decided to use 400 as that's what we have in Figma and newer core components like [SectionNavItem](https://github.com/grafana/grafana/blob/7860ca6c3d7060adb7af33b5d08ad5926fcaecaf/public/app/core/components/PageNew/SectionNavItem.tsx#L84)